Release 1.1 (#9)
vcmi/VCMI.eu: content/news/vcmi-1.1.0-release.md

+++
categories = ["Releases", "Stable"]
date = "2022-12-23"
description = ""
featured = "vcmirelease1_1.jpg"
featuredalt = ""
featuredpath = "date"
linktitle = ""
title = "VCMI 1.1.0 released"
type = "post"
forumurl = "https://forum.vcmi.eu/"

+++

## Big update before 2023!
 
Three months passed since our previous release and we observed many downloads and quite big interest to the project from the community.

> We want to say thank you for that and willing for more! Such support realy motivates us so we were able to increase our efforts towards new features and would like to share with you one more big update this year.

Reviewer: ```suggestion
We want to say thank you for that and willing for more! Such support really motivates us, thus we were able to increase our efforts towards new features and would like to share with you one more big update this year.
```

## iOS support

> VCMI supports one more platfform! Now you can play heroes on your iPad or iPhone and it'll have all game mechanics, including mods. Installation is not so simple as it could be using embedded app store but it definetely worth to make several simple steps from this [instruction](https://wiki.vcmi.eu/Installation_on_iOS) and it will be easier than expected!

Reviewer: ```suggestion
VCMI supports one more platform! Now you can play heroes on your iPad or iPhone and it'll have all game mechanics, including mods. Installation is not so simple as it could be using the official AppStore but it's definitely worth to make several simple steps from this [instruction](https://wiki.vcmi.eu/Installation_on_iOS) and it will be easier than expected!
```

## Online multiplayer

> We overcomed most of challenges on a way to multiplayer so it's finally possible to play VCMI with your friends or with random people at Windows, MacOS, Linux or iOS. You can play over local network or use lobby mode from Launcher to play over internet.

Reviewer: ```suggestion
We have overcome most of challenges on a way to multiplayer so it's finally possible to play VCMI with your friends or with random people at Windows, macOS, Linux or iOS. You can play over local network or use lobby mode from Launcher to play over internet.
```

## Map editor

> Map editor for VCMI was rewritten in order to be cross-platform (currently works on Windows, Mac and Linux) and support mods. Map editor is included into installation package so everyone can create and share their own scenarios.

Reviewer: ```suggestion
Map editor for VCMI was rewritten in order to be cross-platform (currently works on Windows, macOS and Linux) and support mods. Map editor is included into installation package so everyone can create and share their own scenarios.
```

 
## Game engine

> We continue investing into the game stability, tons of bugs were addressed, but don't forget to [report](https://github.com/vcmi/vcmi/issues) about problem if it appears. Version 1.1 has many aesthetics improvements, among them along-awaited ray shooting, tower icons, fixes for battle effects. Music isn't playing from the beginning while switching active screen and you can listen legendary soundtracks in the same way as in original game. Worth to mention that performance of random map generation got severe boost as well.

Reviewer: ```suggestion
We continue investing into the game stability, tons of bugs were addressed, but don't forget to [report](https://github.com/vcmi/vcmi/issues) about problem if it appears. Version 1.1 has many aesthetics improvements, among them long-awaited ray shooting, tower icons, fixes for battle effects. Music isn't playing from the beginning while switching active screen and you can listen to the legendary soundtracks in the same way as in original game. Worth to mention that performance of random map generation got severe boost as well.

**Please note that save games from v1.0 are incompatible with v1.1!**
```

> We are conitinuing investing into game stability, tons of bugs were addressed, but don't forget to [report](https://github.com/vcmi/vcmi/issues) about problem if it appears. Version 1.1 has many aestetics improvements, among them along-awaited ray shooting, tower icons, fixes for battle effects. Music isn't playing from the beggining while switching active screen and you can listen legendary soundtrack in the same way as in original game. Worth to mention that performance of random map generation got severe boost which was quite annoying.

Reviewer 2: I think continue should not be used in present continue form.
```suggestion
We continue investing into the game stability, tons of bugs were addressed, but don't forget to [report](https://github.com/vcmi/vcmi/issues) about problem if it appears. Version 1.1 has many aesthetics improvements, among them along-awaited ray shooting, tower icons, fixes for battle effects. Music isn't playing from the beginning while switching active screen and you can listen legendary soundtracks in the same way as in original game. Worth to mention that performance of random map generation got severe boost as well.
```

 
## Artificial intelligence

> Nullkiller AI in combination with Battle AI is not only more stable but get smarter on the battlefield. Enemy hero may consider winning chances as poor and prefer to escape.

Reviewer 2: ```suggestion
Nullkiller AI in combination with Battle AI is not only more stable but gets smarter on the battlefield. Enemy hero may consider winning chances as poor and prefer to escape.
```

 
## And many others

> Update is not limited with highlighted improvements, almost every game aspect has undergone changes, detailed backlog is on [release page](https://github.com/vcmi/vcmi/releases/tag/1.1.0). Of course, new features and game in common maybe are still not in ideal state but we want people enjoy them and our team will continue doing the best as we can. In turn, we are looking forward for feedback and maybe even contribution into this wonderful project from anyone!

Reviewer 2: ```suggestion
Update is not limited to highlighted improvements, almost every game aspect has undergone some changes, the detailed backlog is on [the release page](https://github.com/vcmi/vcmi/releases/tag/1.1.0). Of course, new features and the game in common are still not in ideal state but we want people enjoy them and our team will continue doing the best we can. In turn, we are looking forward for feedback or even better contribution into this wonderful project from anyone!
```
